fbbt/src/ontology/fbbt-edit.obo

```diff
@@ -56778,6 +56778,7 @@ id: FBbt:00007026
 name: adult midgut epithelium
 def: "Epithelial monolayer of endodermal origin, which lines the adult midgut and which is overlain by the peritrophic membrane." [FlyBase:FBrf0191249]
 subset: cur
+is_a: FBbt:00007027 ! cuboidal/columnar epithelium
```

Comment:
Is all gut epithelium cuboidal/columnar epithelium (as opposed to squamous).  If so, can this classification be moved up?

Reply:
No big deal if not clear.